src/components/organisms/PaneManager/PaneManager.tsx

```diff
@@ -0,0 +1,212 @@
+
+const StyledRow = styled(Row)`
+  background-color: ${BackgroundColors.darkThemeBackground};
+  width: 100%;
+  padding: 0px;
+  margin: 0px;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+`;
+const StyledColumnLeftMenu = styled(Col)`
+  background-color: ${BackgroundColors.darkThemeBackground};
+  height: 100%;
+  padding: 0px;
+  margin: 0px;
+  border-right: ${AppBorders.pageDivider};
+`;
+const StyledColumnPanes = styled(Col)`
+  background-color: ${BackgroundColors.darkThemeBackground};
+  height: 100%;
+  padding: 0px;
+  margin: 0px;
+  overflow-x: visible !important;
+  overflow-y: visible !important;
+`;
+const StyledColumnRightMenu = styled(Col)`
+  background-color: ${BackgroundColors.darkThemeBackground};
+  height: 100%;
+  padding: 0px;
+  margin: 0px;
+  border-left: ${AppBorders.pageDivider};
+`;
+
+const StyledContent = styled(Content)`
+  overflow-y: clip;
+`;
+
+const MenuIcon = (props: {
+  icon: React.ElementType;
+  active: boolean;
+  isSelected: boolean;
+  style?: React.CSSProperties;
+}) => {
+  const {icon: IconComponent, active, isSelected, style: customStyle = {}} = props;
+  const [isHovered, setIsHovered] = useState<boolean>(false);
+
+  const style = {
+    ...customStyle,
+    fontSize: 25,
+    color: Colors.grey7,
+  };
+
+  if (isHovered || (active && isSelected)) {
+    style.color = Colors.grey400;
+  }
+
+  return (
+    <IconComponent style={style} onMouseEnter={() => setIsHovered(true)} onMouseLeave={() => setIsHovered(false)} />
+  );
+};
+
+const iconMenuWidth = 45;
+
+const PaneManager = () => {
+  const dispatch = useAppDispatch();
+  const {windowSize} = useContext(AppContext);
+
+  const contentWidth = windowSize.width - (featureJson.ShowRightMenu ? 2 : 1) * iconMenuWidth;
+  const contentHeight = `${windowSize.height - 75}px`;
+
+  const leftMenuSelection = useAppSelector(state => state.ui.leftMenu.selection);
+  const leftActive = useAppSelector(state => state.ui.leftMenu.isActive);
+  const rightMenuSelection = useAppSelector(state => state.ui.rightMenu.selection);
+  const rightActive = useAppSelector(state => state.ui.rightMenu.isActive);
+
+  const setActivePanes = (side: string, selectedMenu: string) => {
+    if (side === 'left') {
+      if (leftMenuSelection === selectedMenu) {
+        dispatch(toggleLeftMenu());
+      } else {
+        dispatch(setLeftMenuSelection(selectedMenu));
+        if (!leftActive) {
+          dispatch(toggleLeftMenu());
+        }
+      }
+    }
+
+    if (side === 'right' && featureJson.ShowRightMenu) {
+      if (rightMenuSelection === selectedMenu) {
+        dispatch(toggleRightMenu());
+      } else {
+        dispatch(setRightMenuSelection(selectedMenu));
+        if (!rightActive) {
+          dispatch(toggleRightMenu());
+        }
+      }
+    }
+  };
+
+  return (
+    <StyledContent style={{height: contentHeight}}>
+      <StyledRow style={{height: contentHeight + 4}}>
+        <StyledColumnLeftMenu>
```

Comment:
would it make sense to further split this into LeftToolbar, LeftPane, CenterPane, RightPane and RightToolbar components? (they could be in the same file...)